src/modules/admin/admin-tab-categories/AdminTabCategories.scss

```diff
@@ -0,0 +1,43 @@
+@import 'styles/mixins';
+
+.adminTabCategories {
```

Comment:
Add spacing changes - spaces should be consistent with templates search spacing - this is ultra important